cmake/Dependencies.cmake

```diff
@@ -397,6 +397,28 @@ if(USE_CUDA)
   endif()
 endif()
 
+# ---[ HIP
+if(USE_HIP)
+  include(cmake/public/LoadHIP.cmake)
+  if(PYTORCH_FOUND_HIP)
+    message(INFO "Compiling with HIP for AMD.")
+
+    set(Caffe2_HIP_CXX_FLAGS "-D__HIP_PLATFORM_HCC__=1")
+    set(Caffe2_HIP_INCLUDES
+      ${hip_INCLUDE_DIRS} ${rocrand_INCLUDE_DIRS} ${hiprand_INCLUDE_DIRS} ${rocblas_INCLUDE_DIRS} ${miopen_INCLUDE_DIRS} ${Caffe2_HIP_INCLUDES} ${thrust_INCLUDE_DIRS})
+    set(Caffe2_HIP_DEPENDENCY_LIBS
+      ${rocrand_LIBRARIES} ${hiprand_LIBRARIES} ${PYTORCH_HIP_HCC_LIBRARIES} ${PYTORCH_MIOPEN_LIBRARIES})
```

Comment:
Can use miopen_LIBRARIES and hip_hcc_LIBRARIES here.

Reply:
See my comments in cmake/public/LoadHIP.cmake